analyzer/src/funTest/assets/projects/external/jgnash-core-expected-output.yml

```diff
@@ -10,7 +10,7 @@ project:
   aliases: []
   vcs:
     type: "git"
-    url: "git://github.com:ccavanaugh/jgnash.git/jgnash-core"
+    url: "git://github.com:ccavanaugh/jgnash.git"
```

Comment:
`vcs_processed` didn't have `jgnash-core` in the URL because it prefers information from the working tree. It has `jgnash-core` as path though, which is correct.